### PR TITLE
luci-app-udpxy: fix luci-app-udpxy

### PR DESCRIPTION
**Pull request by findlayfeng**
- [x] This PR is not from my *main* or *master* branch :poop:, but a *separate* branch :white_check_mark:
- [x] Each commit has a valid :black_nib: `Signed-off-by: <my@email.address>` row (via `git commit --signoff`)
- [x] Each commit and PR title has a valid :memo: `<package name>: title` first line subject for packages
- [x] Incremented :up: any `PKG_VERSION` in the Makefile
- [x] Tested on: (architecture, openwrt version, browser) :white_check_mark:
- [x] \( Preferred ) Mention: @1715173329 
- [ ] \( Preferred ) Screenshot or mp4 of changes:
- [ ] \( Optional ) Closes: e.g. openwrt/luci#issue-number
- [x] \( Optional ) Depends on: e.g. openwrt/packages#24702 in sister repo
- [ ] Description: (describe the changes proposed in this PR)

### CODE REVIEW

**Comment by findlayfeng** [comment]
@1715173329

File: applications/luci-app-udpxy/htdocs/luci-static/resources/view/udpxy.js [hunk 1]
```diff
@@ -27,18 +28,24 @@ return view.extend({
 
 		o = s.option(form.Flag, 'status', _('Client statistics'));
 
-		o = s.option(form.Value, 'bind', _('HTTP Listen interface'));
-		o.datatype = 'or(ipaddr, network)';
-		o.placeholder = '0.0.0.0 || lan1';
+		o = s.option(widgets.DeviceSelect, 'bind', _('HTTP Listen interface'));
```
Comment (1715173329) [inline applications/luci-app-udpxy/htdocs/luci-static/resources/view/udpxy.js:31]: It's clear that what you did is more than "add source_network option".
Using `widgets.DeviceSelect` here can be misleading as it's only for interface. Though you can put a ipaddr you can see a confusing prefix "custom interface:".

File: applications/luci-app-udpxy/po/en/udpxy.po [hunk 2]
```diff
@@ -1,103 +1,102 @@
 msgid ""
-msgstr ""
```
Comment (1715173329) [inline applications/luci-app-udpxy/po/en/udpxy.po:2]: update translation files with `./build/i18n-update.pl applications/luci-app-udpxy/po` or just leave it as-is.

**Comment by systemcrash** [comment]
@1715173329 is this OK for you?

**Comment by 1715173329** [comment]
> @1715173329 is this OK for you?

I'm not the maintainer of this package nor an expert on luci, so it's up to you ;)

**Pull request merged**